src/config/default-config.js

```diff
@@ -10,6 +11,16 @@ const logger = getLogger();
 // If we are, assign NODE_ENV based on the GitLab (AWS/k8s cluster) environment.
 // If NODE_ENV is set, that will take precedence over the GitLab
 // environment.
```

Comment:
this comment is very, very outdated and looks like it's out of place. Can you move it where it should be and changes to references to Gitlab with Github?